src/landingPage/LandingPage.js

```diff
@@ -4,15 +4,27 @@ import News from "./News";
 import * as s from "./LandingPage.sc.js";
 import Contributors from "./Contributors";
 import { Redirect } from "react-router-dom";
+import React, { useState } from "react";
+import LanguageSettings from "../components/LanguageSettings";
 
 export default function LandingPage() {
+  const forceUpdateHook = useState(0)[1];
+
+  localStorage.setItem("systemLanguage", JSON.stringify({
+    name: "Dansk",
+    code: "da",
+  }));
   if (LocalStorage.hasSession()) {
     return <Redirect to={{ pathname: "/articles" }} />;
   }
+  
   return (
     <div>
       <s.LoginHeader>
         <s.HeaderTitle>Zeeguu</s.HeaderTitle>
+        <LanguageSettings
+          useForceUpdate={() => forceUpdateHook((value) => value + 1)}
```

Comment:
Could you explain what `() => forceUpdateHook((value) => value + 1)` does?

Reply:
@bwesth - is this forceUpdateHook defined by you?
I guess something like this would make it more clear? 
https://www.npmjs.com/package/use-force-update-hook

Reply:
> Could you explain what `() => forceUpdateHook((value) => value + 1)` does?

Very simply it tells React that something has changed in the componet it is used in, and therefore forces a rerender. This was due to a problem, that the language was dependant on static imports, and did therefore not change when the strings language changed. Does that make sense?

Reply:
> @bwesth - is this forceUpdateHook defined by you?
> I guess something like this would make it more clear?
> https://www.npmjs.com/package/use-force-update-hook

@mircealungu 
This introduces another dependency. Would you rather I used the package than making my own? My own version looks like this in /landingPage/LandingPage.js:

11  const [,forceUpdateHook] = useState(0);
...
26 useForceUpdate={() => forceUpdateHook((value) => value + 1)}

Very simple, very short and compact. Not very readable.